src/applications/resources-and-support/components/SearchBar.jsx

```diff
@@ -84,7 +85,7 @@ export default function SearchBar({
             ? `${searchSettings.rootUrl}/`
             : `${resourcesSettings.rootUrl}/`
         }
-        onSubmit={isGlobalSearch ? null : onSubmit}
+        onSubmit={onSubmit}
```

Comment:
Nice Change!

Reply:
ya definitely didn't notice this when I sent up the latest analytics PR 😂 eek!